app/fieldmapping.py

```diff
@@ -88,6 +88,18 @@ def map_employee_history_job_role_table_headers(
 
   return table_generation(mapping_entries)
 
+def map_full_address_details(employee_info):
+  complete_address = ''
+  address_parts  = [
+      'address1',
+      'address2',
+      'town',
+      'country',
+      'postcode',
+      ]
+  for address_part in address_parts:
+    complete_address = complete_address + str( employee_info.get(address_part,'') ) + ",  "
```

Comment:
The logic here seems odd, and will leave a trailing comma+space at the end of the joining.
Python provides a method for this kind of situation: `string.join(iterable)`
In this case, `complete_address = ", ".join(part + str(employee_info.get(part, '') for part in address_parts)` might give you the full string in one line.